tests/redis/test_manager.py

```diff
@@ -80,6 +80,13 @@ async def test_cache_manager_save_and_get_many_objects_with_keys_success(cache_m
     assert response == test_dtos
 
 
+async def test_cache_manager_save_and_get_many_objects_with_keys_as_it_is_success(cache_manager: CacheManager) -> None:
```

Comment:
It would be great to have another test where you don't use the `object_class` argument, but some simple data types.